docs/asciidoc/coreFeatures.adoc

```diff
@@ -173,7 +173,7 @@ immediate scheduling, etc...).
 The https://projectreactor.io/docs/core/release/api/reactor/core/scheduler/Schedulers.html[`Schedulers`]
 class has static methods that give access to the following execution contexts:
 
- * The current thread (`Schedulers.immediate()`).
+ * The same thread that is currently processing the chain (`Schedulers.immediate()`, can be seen as a "null object").
```

Comment:
maybe we can explain it as a "no-op" scheduler?

Otherwise, people may think that they can "capture" the current thread and use it later as a Scheduler

Reply:
ok yeah let's try to avoid that! rephrased.